feature-libs/organization/.release-it.json

```diff
@@ -27,7 +27,8 @@
             "peerDependencies.@spartacus/core",
             "peerDependencies.@spartacus/storefront",
             "peerDependencies.@spartacus/schematics",
-            "peerDependencies.@spartacus/user"
+            "peerDependencies.@spartacus/user",
+            "peerDependencies.@spartacus/cart"
```

Comment:
I guess this is caused by `Principal`.

Reply:
Not just for `Principal`.  From just the model files in order lib, these 6 cart models are imported:

```
  DeliveryOrderEntryGroup,
  PaymentDetails,
  PaymentType,
  Principal,
  PromotionResult,
  Voucher,

```

Reply:
maybe depends on `order`? I just feel organization depends on cart not make sense.

Reply:
But, from the changed files, I only see `Principal`. I think maybe we can leave `Principal` into core, or move it to user in future. I think `organization` depends on `cart` not make sense.

Reply:
Agreed, we leave Principal in core